dynamixels: ['head_pan', 'head_tilt', 'arm_pan', 'arm_tilt', 'elbow_tilt', 'elbow_roll', 'wrist_tilt', 'wrist_pan', 'gripper']




head_tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_tilt_joint
    joint_speed: 1.17
    motor:
        id: 4
        init: 512
        min: 300
        max: 800

head_pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_pan_joint
    joint_speed: 1.17
    motor:
        id: 9
        init: 512
        min: 300
        max: 800


arm_pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_pan_joint
    joint_speed: 1.17
    motor:
        id: 3
        init: 512
        min: 0
        max: 1024





arm_tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController 
    joint_name: arm_tilt_joint
    joint_speed: 1.17
    motor:
        id: 10
        init: 512
        min: 717
        max: 126
    


elbow_tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController 
    joint_name: elbow_tilt_joint
    joint_speed: 1.17
    motor:
        id: 8
        init: 512
        min: 881
        max: 126

elbow_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController 
    joint_name: elbow_roll_joint
    joint_speed: 1.17
    motor:
        id: 7
        init: 512
        min: 881
        max: 126
   


wrist_tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_tilt_joint
    joint_speed: 1.17
    motor:
        id: 5
        init: 512
        min: 717
        max: 126

wrist_pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_pan_joint
    joint_speed: 1.17
    motor:
        id: 6
        init: 512
        min: 717
        max: 126

gripper_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: gripper_joint
    joint_speed: 1.17
    motor:
        id: 2
        init: 512
        min: 717
        max: 126

right_arm_controller:
   controller:
       package: dynamixel_controllers
       module: joint_trajectory_action_controller
       type: JointTrajectoryActionController
   joint_trajectory_action_node:
       min_velocity: 0.25
       constraints:
           goal_time: 1.0

head_traj_controller:
   controller:
       package: dynamixel_controllers
       module: joint_trajectory_action_controller
       type: JointTrajectoryActionController
   joint_trajectory_action_node:
       min_velocity: 0.25
       constraints:
           goal_time: 1.0


